docs/src/docs/asciidoc/index.adoc

```diff
@@ -19,6 +19,20 @@ If you are using alternative build systems, see <<alternative-build-systems.adoc
 [[changelog]]
 == Changelog
 
+=== Release 0.9.23
+
```

Comment:
There are more changes that these: https://github.com/graalvm/native-build-tools/compare/0.9.22...master